ydb/docs/en/core/reference/ydbops/rolling-restart-scenario.md

```diff
@@ -0,0 +1,91 @@
+## Performing {{ ydb-short-name }} cluster restart using ydbops
+
+{% note info %}
+
+The article is being updated. Expect new content to appear and minor fixes to existing content.
+
+{% endnote %}
+
+`ydbops` can be used to perform the rolling restart operation: restarting all (or a subset) of {{ ydb-short-name }} cluster nodes while maintaining cluster availability.
+
+The subcommand responsible for this operation is `ydbops restart`.
+
+## General algorithm
+
+There are multiple options to `ydbops restart` that act as filters. Filters are implicitly connected with and `AND` condition, meaning that if you supply multiple filters, only the nodes that satisfy all of them at once will be targeted.
+
+Therefore, specifying no filters at all will target all nodes for restart.
+
+There are two special filters that are an exception to these rule: `--storage` and `--tenant`: 
+- specifying only `--storage` will filter storage nodes only;
+- specifying only `--tenant` will filter tenant nodes only;
+- however, specifying them both is `OR`-ed, selects all nodes and is equivalent to not specifying these two filters at all.
+
+The algorithm will always work in two phases:
+- first, it will determine whether any storage nodes fit the restart filters and restart these nodes only.
+- after all storage nodes have been restarted (or it has been determined that no storage nodes are selected), the process repeats for tenant nodes.
+
+## Examples
+
+The following examples assume you have also specified all the required connection options (such as endpoint or credentials).
+
+### Restarting all nodes in the cluster
+
+The invocation would restart all the nodes in the cluster: all storage nodes first, followed by all tenant nodes.
+
+```
+ydbops restart 
+```
+
+### Restarting only storage\tenant nodes
+
+It is possible to restart storage nodes only:
+
+```
+ydbops restart --storage
+```
+
+Or tenant nodes only:
+
+```
+ydbops restart --tenant
+```
+
+Additionally, only specific tenants may be restarted by specifying `--tenant-list`:
+
+```
+ydbops restart --tenant-list=/domain/database_name_1,/domain/database_name_2
+```
+
+### Restarting only specific nodes
+
+It is possible to restart only specific nodes by supplying FQDNs with a `--hosts` option:
+
+```
+ydbops restart --hosts=node1.some.zone,node2.some.zone
+```
+
+Or by supplying node ids directly:
+
+```
+ydbops restart --hosts=1,2,3
+```
+
+### Restarting based on uptime
+
+It is possible to restart only the nodes that have specific uptime by using `--started` option:
+
+```
+ydbops restart --started >2024-03-13T17:00:00Z
```

Comment:
It looks like some quotes are missing. Currently, it seems like it will pipe the command output to "2024-03-13T17:00:00Z".